kitsune/users/tests/test_api.py

```diff
@@ -327,3 +328,40 @@ def test_request_password_reset(self, get_current):
         res = self.client.get(url)
         eq_(res.status_code, 204)
         eq_(1, len(mail.outbox))
+
+    def test_create_user_no_email(self):
+        # There is at least one user in existence due to migrations
+        number_users = User.objects.count()
+
+        url = reverse('user-list')
+        res = self.client.post(url, {
+            'username': 'kris',
+            'password': 'testpass'
+        })
+
+        eq_(res.status_code, 201)
+        eq_(User.objects.count(), number_users + 1)
+        u = User.objects.order_by('-id')[0]
+        eq_(u.username, 'kris')
+        eq_(u.is_active, True)
+
+    def test_create_user_with_email(self):
+        # There is at least one user in existence due to migrations
+        number_users = User.objects.count()
+
+        username = 'kris-{}'.format(random())
+        url = reverse('user-list')
+        res = self.client.post(url, {
+            'username': username,
+            'password': 'testpass',
+            'email': 'kris@example.com'
+        })
+
+        import q
```

Comment:
I've thought about writing that hook too, but it inevitably ends up harder than I want it to be. I think it would be good to have a directory of hooks system eventually, but that is hard and non-standard. Another idea: I think one of pep8, pyflakes, or flake8 supports some sort of plugin interface, which would probably be a good way to make this. I think this is how mccabe works.

Or I could just do better at not committing this...